src/CircleCI-MinGW.sh

```diff
@@ -38,8 +38,8 @@ mv ../run/john ../run/john.exe
 
 # the mingw build does not name many exe files correctly.  Fix that.  Also strip all exe files for distro.
 cd ../run
-for f in `ls -l | grep wxr | grep -v [\.][epr] | cut -c 46-` ; do mv $f $f.exe ; done
-for f in benchmark-unify mailer makechr relbench ; do mv $f.exe $f ; done
+# for f in `ls -l | grep wxr | grep -v [\.][epr] | cut -c 46-` ; do mv $f $f.exe ; done
+# for f in benchmark-unify mailer makechr relbench ; do mv $f.exe $f ; done
 for f in *.exe ; do x86_64-w64-mingw32-strip $f ; done
```

Comment:
This for loop was generating a lot of error messages on my local Fedora 24 box. This whole thing is indeed very unsafe.